Tests/scs-test-runner.py

```diff
@@ -152,8 +199,9 @@ def _move_file(source_path, target_path):
 @click.option('--num-workers', 'num_workers', type=int, default=5)
 @click.option('--monitor-url', 'monitor_url', type=str, default=MONITOR_URL)
 @click.option('-o', '--output', 'report_yaml', type=click.Path(exists=False), default=None)
+@click.option('--upload/--no-upload', default=True)
```

Comment:
NO! This has nothing to with the issue at hand.

Also: why? I specifically told you we don't want people to cherry-pick their results! If you really don't want to upload, then supply `--monitor-url localhost` (or something unreachable)

Reply:
reverted it